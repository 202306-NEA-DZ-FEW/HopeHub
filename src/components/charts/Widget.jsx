import React from "react";

function Widget({ title, value }) {
    return (
        <div className='w-48 h-28 rounded-lg p-4 flex flex-col shadow-[0px_0px_8px_2px_rgba(0,0,0,10%)]'>
            <span className='text-2xl font-bold text-NeutralBlack'>
                {title}
            </span>
            <span className='text-5xl font-semibold text-Accent'>{value}</span>
        </div>
    );
}

export default Widget;
