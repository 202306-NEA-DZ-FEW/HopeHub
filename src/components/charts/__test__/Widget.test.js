import renderer from "react-test-renderer";

import Widget from "../Widget";

it("renders correctly", () => {
    const tree = renderer.create(<Widget />).toJSON();
    expect(tree).toMatchSnapshot();
});
