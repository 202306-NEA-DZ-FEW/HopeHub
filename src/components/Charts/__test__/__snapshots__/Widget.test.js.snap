// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-48 h-28 rounded-lg p-4 flex flex-col shadow-[0px_0px_8px_2px_rgba(0,0,0,10%)]"
>
  <span
    className="text-2xl font-bold text-NeutralBlack"
  />
  <span
    className="text-5xl font-semibold text-Accent"
  />
</div>
`;
